atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -308,10 +309,11 @@ def get_image_output(self):
 
     def get_digests(self):
         """
-        Returns a map of images to their digests
+        Returns a map of images to their digests and map of media types to their images
         """
 
         digests = {}  # repository -> digests
+        docker_digests = {}
```

Comment:
Can we call it `typed_digests` instead of `docker_digests` maybe? Also would be nice to have a comment, `# media type -> digest`. While here I notice that the line above should probably say "image -> digests" (not repository).

Reply:
Fixed.